components/text/hunspell/Makefile

```diff
@@ -11,21 +11,22 @@
 #
 # Copyright (c) 2016 Alexander Pyhalov
 # Copyright (c) 2017 Ken Mays
+# Copyright (c) 2023 Klaus Ziegler
 #
 
+PREFERRED_BITS=64
 BUILD_BITS = 32_and_64
```

Comment:
I think `BUILD_BITS = 64_and_32` should do the same without the `PREFERRED_BITS`.

Reply:
Correct